Action.c

```diff
@@ -432,48 +450,50 @@ static Htop_Reaction actionTogglePauseProcessUpdate(State* st) {
 static const struct {
    const char* key;
    const char* info;
+   bool roInactive;
 } helpLeft[] = {
-   { .key = " Arrows: ", .info = "scroll process list" },
-   { .key = " Digits: ", .info = "incremental PID search" },
-   { .key = "   F3 /: ", .info = "incremental name search" },
-   { .key = "   F4 \\: ",.info = "incremental name filtering" },
-   { .key = "   F5 t: ", .info = "tree view" },
-   { .key = "      p: ", .info = "toggle program path" },
-   { .key = "      m: ", .info = "toggle merged command" },
-   { .key = "      Z: ", .info = "pause/resume process updates" },
-   { .key = "      u: ", .info = "show processes of a single user" },
-   { .key = "      H: ", .info = "hide/show user process threads" },
-   { .key = "      K: ", .info = "hide/show kernel threads" },
-   { .key = "      F: ", .info = "cursor follows process" },
-   { .key = "  + - *: ", .info = "expand/collapse tree/toggle all" },
-   { .key = "N P M T: ", .info = "sort by PID, CPU%, MEM% or TIME" },
-   { .key = "      I: ", .info = "invert sort order" },
-   { .key = " F6 > .: ", .info = "select sort column" },
+   { .key = " Arrows: ",  .roInactive = false, .info = "scroll process list" },
+   { .key = " Digits: ",  .roInactive = false, .info = "incremental PID search" },
+   { .key = "   F3 /: ",  .roInactive = false, .info = "incremental name search" },
+   { .key = "   F4 \\: ", .roInactive = false, .info = "incremental name filtering" },
+   { .key = "   F5 t: ",  .roInactive = false, .info = "tree view" },
+   { .key = "      p: ",  .roInactive = false, .info = "toggle program path" },
+   { .key = "      m: ",  .roInactive = false, .info = "toggle merged command" },
+   { .key = "      Z: ",  .roInactive = false, .info = "pause/resume process updates" },
+   { .key = "      u: ",  .roInactive = false, .info = "show processes of a single user" },
+   { .key = "      H: ",  .roInactive = false, .info = "hide/show user process threads" },
+   { .key = "      K: ",  .roInactive = false, .info = "hide/show kernel threads" },
+   { .key = "      F: ",  .roInactive = false, .info = "cursor follows process" },
+   { .key = "  + - *: ",  .roInactive = false, .info = "expand/collapse tree/toggle all" },
+   { .key = "N P M T: ",  .roInactive = false, .info = "sort by PID, CPU%, MEM% or TIME" },
+   { .key = "      I: ",  .roInactive = false, .info = "invert sort order" },
+   { .key = " F6 > .: ",  .roInactive = false, .info = "select sort column" },
```

Comment:
Spacing, order of fields should match declaration

Reply:
reordered